boards/arm/arty/Kconfig.board

```diff
@@ -6,3 +6,8 @@
 config BOARD_ARTY_A7_ARM_DESIGNSTART_M1
 	bool "Digilent Arty A7 ARM DesignStart Cortex-M1"
 	depends on SOC_SERIES_ARM_DESIGNSTART
+
+config BOARD_ARTY_A7_ARM_DESIGNSTART_M3
+	bool "Digilent Arty A7 ARM DesignStart Cortex-M3"
+	select CPU_HAS_ARM_MPU
```

Comment:
@henrikbrixandersen 
Thinking more about this: would be more elegant if you move it in Kconfig.defconfig, under the definition of CPU_CORTEX_M_HAS_SYSTICK and make it default y if the M3 BOARD is used.

That's more for having both _HAS_ Kconfig options in one place; it should not change anything.

Reply:
@ioannisg Thanks, that's a bit more clean. Done.